### PR TITLE
[Intl] Add a special locale to strip emojis

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Resolves https://github.com/symfony/symfony-docs/issues/17607

### CODE REVIEW

**Review by alamirault — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Thanks Alexandre!

File: components/string.rst [hunk 1]
```diff
@@ -552,6 +552,17 @@ from GitHub or Slack, use the first argument of ``withEmoji()`` method::
...
+If you want to strip emojis from a string, you can also use the special
+``strip`` locale::
+
+    use Symfony\Component\String\Slugger\AsciiSlugger;
+
+    $slugger = new AsciiSlugger();
+    $slugger = $slugger->withEmoji('strip');
+
+    $slug = $slugger->slug('a 😺, 🐈‍⬛, and a 🦁');
+    // $slug = 'a-and-a';
```
Comment (OskarStark) [inline components/string.rst:564]: versionadded missing, or did you add one while merging @javiereguiluz ?

Cannot check I am currently on a phone 📱

Comment (alexandre-daubois) [reply components/string.rst:564]: It's been added in https://github.com/symfony/symfony-docs/commit/874f0260708961aad282888ce2ade08e5a6ccada, thanks for this Javier!

Comment (OskarStark) [reply components/string.rst:564]: Thanks for the feedback

**Pull request merged**